frontend/jupyter/src/app/pages/form/form-new/volume/existing/pvc/pvc.component.ts

```diff
@@ -26,7 +26,7 @@ export class ExistingPvcComponent implements OnInit {
 
   ngOnInit(): void {
     this.ns.getSelectedNamespace().subscribe(ns => {
-      this.backend.getPVCs(ns).subscribe(pvcs => {
+      this.backend.getVolumes(ns).subscribe(pvcs => {
```

Comment:
Just a note, volume vs pvc

Reply:
Yes I have noted this in the testing ticket